src/main/java/com/team4/testingsystem/converters/notifications/TestDeassignedNotificationConverter.java

```diff
@@ -0,0 +1,24 @@
+package com.team4.testingsystem.converters.notifications;
+
+import com.team4.testingsystem.dto.NotificationDTO;
+import com.team4.testingsystem.entities.Notification;
+import com.team4.testingsystem.enums.NotificationType;
+import org.springframework.stereotype.Component;
+
+@Component
+public class TestDeassignedNotificationConverter implements SingleNotificationConverter {
+    @Override
+    public NotificationDTO convertToDTO(Notification notification) {
+        return NotificationDTO.builder()
+                .id(notification.getId())
+                .type(NotificationType.TEST_DEASSIGNED)
```

Comment:
Seems like we can create a single builder and just provide the notificationType

Reply:
I've extracted this part to a single method in abstract class